full/build.gradle

```diff
@@ -48,7 +47,7 @@ dependencies {
 //    implementation group: 'commons-codec', name: 'commons-codec', version: '1.14'
     implementation group: 'com.jayway.jsonpath', name: 'json-path', version: '2.4.0'
     implementation group: 'org.hdrhistogram', name: 'HdrHistogram', version: '2.1.9'
-    implementation group: 'org.neo4j.driver', name: 'neo4j-java-driver', version: '4.0.0'
+    compileOnly group: 'org.neo4j.driver', name: 'neo4j-java-driver-slim', version: '4.4.3'
```

Comment:
Make the driver compile only in other branches to avoid overwriting the one in the database when we plug in apoc

Reply:
What do you mean with 'only in other branches'? I'm not sure I follow...